.pipelines/singletenancy/cilium-dualstack-overlay/cilium-dualstackoverlay-e2e-step-template.yaml

```diff
@@ -0,0 +1,277 @@
+parameters:
+  name: ""
+  testDropgz: ""
+  clusterName: ""
+  testHubble: false
+
+steps:
+  - bash: |
+      echo $UID
+      sudo rm -rf $(System.DefaultWorkingDirectory)/*
+    displayName: "Set up OS environment"
+
+  - checkout: self
```

Comment:
What all is this doing?

Reply:
IMO these two steps are redundant and excessive. AZP already check(s)out self by default and deleting the working directory does not do anything productive. What is it that you are trying to achieve?

Reply:
It is very possible that this step is unnecessary, and we can delete it. I mostly just added it in since we have been doing this in the other cilium templates as well.

Reply:
I'll just go ahead and remove it from here if it is unnecessary and we can also remove it from the other cilium tests too if we want later.